sys/dev/random/random_harvestq.c

```diff
@@ -289,15 +329,22 @@ random_print_harvestmask_symbolic(SYSCTL_HANDLER_ARGS)
 {
 	struct sbuf sbuf;
 	int error, i;
+	bool first;
```

Comment:
One last nitpick: There should be a blank line between variable declarations and code.